mantle/go.mod

```diff
@@ -1,56 +1,107 @@
 module github.com/coreos/mantle
 
-go 1.12
+go 1.17
```

Comment:
This is probably worth factoring out as a separate preparatory commit.

Reply:
Do you mean a preparatory PR, or just creating a dedicated commit with the vendoring in a follow-up commit to make it more visible?

Reply:
Separate commit in the same PR should be fine.